src/screens/WebsocketResearchScreen/SampleChatScreen.tsx

```diff
@@ -68,34 +82,47 @@ const SampleChatScreen = () => {
     flatlistRef.current?.scrollToEnd();
   };
 
+  React.useEffect(() => {
+    return () => {
+      clearReplyPreview();
+    };
+  }, []);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This `useEffect` hook ensures that `clearReplyPreview` is called when the component unmounts. This is a good practice as it helps in cleaning up any leftover state related to the reply preview when the chat screen is closed.



<!-- This is an auto-generated comment by OSS CodeRabbit -->